lib/node_modules/@stdlib/number/float32/base/significand/README.md

```diff
@@ -91,6 +91,93 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/number/float32/base/significand.h"
+```
+
+#### stdlib_base_float32_significand( x )
+
+
+Returns an integer `y` corresponding to the significand of a single-precision floating-point number `x`.
+
+```c
+#include <stdint.h>
+int32_t y;
+stdlib_base_float32_significand( 3.14 );
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] float` input value.
+-   **exp**: `[out] int32_t*` Output value.
+
+```c
+void stdlib_base_float32_significand( const float x );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/number/float32/base/significand.h"
+#include <stdint.h>
+#include <stdio.h>
+int main() {
+    float x[] = { 1.0, 3.14, 0.0, -0.0, 3.14e-308, 3.14e+308, 1.0/0.0, 0.0/0.0 };
+    int32_t y;
+    int i;
+    for ( i = 0; i < 8; i++ ) {
+        y = stdlib_base_float32_significand( x[ i ] );
+        printf( "x: %f. significand: %d.\n", x[ i ], y );
```

Comment:
Instead of `%d`, prob better to use inttypes and the associated format specifier.